docs/ru/engines/database-engines/materialized-postgresql.md

```diff
@@ -0,0 +1,66 @@
+---
+toc_priority: 30
+toc_title: MaterializedPostgreSQL
+---
+
+# MaterializedPostgreSQL {#materialize-postgresql}
```

Comment:
Can you please also add intro description same as in MaterializeMySQL

Reply:
> Can you please also add intro description same as in MaterializeMySQL

Ok, I will do it

Reply:
```suggestion
# [экспериментальный] MaterializedPostgreSQL {#materialize-postgresql}
```